mw-webapp/src/service/WayService.ts

```diff
@@ -88,9 +111,27 @@ export class WayService {
   /**
    * Get WaysDTO
    */
-  public static async getWaysDTO(lastWayUuid?: string): Promise<WayDTO[]> {
+  public static async getWaysDTO(lastWayUuid?: string, filter?: GetWaysFilter): Promise<WayDTO[]> {
```

Comment:
its super hard to work with such interface, use object instaedd of two optional args